### DISCUSSION TITLE
crear repositorio

### DISCUSSION THREAD

**Discussion by maralejandram** [community/community#54532] [New to GitHub]
Hola, Intento crear un nuevo repositorio, pero cuando voy a poner el nombre me salta error y me saca de la pagina

**Comment by ingridinara** [comment]
Hola Mara, has conseguido crear tu repositorio ? Lo estas teniendo problema al crear el repo directamente en la web del github ( en este caso a principio solo tienes que ir a `https://github.com/EscribesTuNombreEnGitHub`  y luego tienes que ir a **repositorio** y **nuevo** ) o estas haciendo desde la linea de comando (en este caso tienes que crear la carpeta CD NombreDeLaCarpeta y en seguida iniciar el repo en tu PC con git init  )? Seria mas facil intentar ayudarte si subes un print del error que te da. un saludo!

**Reply by maralejandram** [reply]
Hola, ya lo pude crear, pero se ve que no acepta cualquier nombre. gracias por responder!
